ouroboros-network/testlib/Ouroboros/Network/BlockFetch/Examples.hs

```diff
@@ -148,8 +148,8 @@ blockFetchExample0 fetchMode decisionTracer clientStateTracer clientMsgTracer
           })
         >> return ()
 
-    headerForgeUTCTime (FromConsensus x) =
-        pure $ convertSlotToTimeForTestsAssumingNoHardFork (blockSlot x)
+    headerForgeUTCTime =
+        convertSlotToTimeForTestsAssumingNoHardFork . headerSlot
```

Comment:
Phew, it was a non-trivial effort to convince myself that `headerSlot` is indeed equivalent to `blockSlot` here :detective: